manager-bundle/composer.json

```
@@ -31,7 +31,7 @@
         "ext-json": "*",
         "contao/core-bundle": "self.version",
         "contao/manager-plugin": "^2.4",
-        "doctrine/dbal": "^3.6",
+        "doctrine/dbal": "^3.6 || ^4.0",
```

```suggestion
        "doctrine/dbal": "^3.6 || 4.3.x-dev",
```
Or was this different on purpose?

---

It was on purpose - but in the end it does not really matter. Technically only the `core-bundle` needs at least `^4.3`.